src/components/tracked-controls.js

```diff
@@ -18,7 +17,13 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    hand: {type: 'string', default: 'right'},
```

Comment:
I don't remember picking daydream handedness, but may be forgetting.

for vive and oculus, gamepad API does expose hand (although originally vive hand value was unreliable so wasn't used; might be able to change that now)

I don't think gamepad API provides hand for daydream, so maybe this would be defaultHand (to use when gamepad API does not specify) ?

I do think we need headElement, unless there is some better way to automatically select the head element/entity to use with arm model?

I don't think you can actually use 3dof since it starts with a number, but maybe three-dof-controls (and/or zero-dof-controls)... the trick there being that tracked-controls needs to avoid updating pose, because another component is going to do it.